app/kumactl/cmd/inspect/inspect_dataplanes_test.go

```diff
@@ -31,7 +31,7 @@ type testDataplaneOverviewClient struct {
 	overviews    []*mesh_core.DataplaneOverviewResource
 }
 
-func (c *testDataplaneOverviewClient) List(_ context.Context, _ string, tags map[string]string) (*mesh_core.DataplaneOverviewResourceList, error) {
+func (c *testDataplaneOverviewClient) List(_ context.Context, _ string, tags map[string]string, gatewayDataPlaneFilter bool) (*mesh_core.DataplaneOverviewResourceList, error) {
```

Comment:
how about just `gateway`? `tags` is just `tags`, not `tagsFilter`

The alternative  solution is to introduce Filter struct with fields `tags` and `gateway`

Reply:
there is no tagsFilter

Reply:
Yes, that's my point - `gatewayDataPlaneFilter` should be just `gateway` since `tags` are just `tags`, not `tagsFilter`.